FrontEnd/src/components/adminPage/detail-view/DeleteModal.jsx

```diff
@@ -1,9 +1,15 @@
 import css from './DeleteModal.module.css';
+import PropTypes from 'prop-types';
 
-function DeleteModal({active, setActive}) {
+function DeleteModal({ active, setActive, onDelete }) {
 
-    const onDeleteClick = () => {
-        console.log('Delete');
+    const onDeleteClick = async () => {
+        try {
+            await onDelete();
+            setActive(false);
+        } catch (error) {
+            console.error('Failed to delete:', error);
+        }
```

Comment:
Could we use `swr` mutation here?

Reply:
There are no requests here, it's just a double confirmation window